pkg/stanza/operator/input/windows/input.go

```diff
@@ -216,6 +216,16 @@ func (i *Input) read(ctx context.Context) int {
 	events, err := i.subscription.Read(i.maxReads)
 	if err != nil {
 		i.Logger().Error("Failed to read events from subscription", zap.Error(err))
+		if i.isRemote() && (err.Error() == "The handle is invalid." || err.Error() == "subscription handle is not open") {
```

Comment:
```suggestion
		if i.isRemote() && (errors.Is(err, windows.ERROR_INVALID_HANDLE) || err.Error() == "subscription handle is not open") {
```

Or, maybe extract `subscription handle is not open` into a new variable and do 

```suggestion
		if i.isRemote() && (errors.Is(err, windows.ERROR_INVALID_HANDLE) || errors.Is(err, errHandleNotOpen)) {
```